challenge.js

```diff
@@ -42,6 +42,7 @@
  */
 
 // ANSWER
+grandparentDiv.getElementsByTagName("div")
```

Comment:
`getElementsByTagName` is a method of `document`, not `Element` (the class of an HTML element object). This question wanted you to write a CSS selector string to pass to `document.querySelectorAll` that would select E (among other elements). Here is an explanation with a solution:

Using only tag names and parent/child relationships, here is a solution:

`body > div > p`

`>` is the immediate descendant operator in CSS. So we're looking for a `p` that is the child of a `div`, which is the child of the `body`.

Note that this won't necessarily select *only* E.